Maths/AverageMedian.js

```diff
@@ -9,22 +9,14 @@
  */
 
 const averageMedian = (sourceArrayOfNumbers) => {
-  let numbers = [...sourceArrayOfNumbers]
-  let median = 0
+  const numbers = [...sourceArrayOfNumbers].sort(sortNumbers)
   const numLength = numbers.length
-  numbers = numbers.sort(sortNumbers)
 
   if (numLength % 2 === 0) {
```

Comment:
Could make this a ternary even.

Reply:
yes sure i totally forgot that we can do that way too